docs/get-started/HA.md

```diff
@@ -0,0 +1,666 @@
+| `PMM_PASSIVE_NODE_ID`                                  | The unique ID for your first passive PMM server node.</br></br>Example: `pmm-server-passive`
+| `PMM_PASSIVE2_IP`                                         | The IP address of the instance where the second passive PMM server is running or the desired IP address for your second passive PMM server container within the Docker network, depending on your setup.</br></br>Example: `17.10.1.7`
+| `PMM_PASSIVE2_NODE_ID`                                    | The unique ID for your second passive PMM server node.</br></br>Example: `pmm-server-passive2`
+| `PMM_DOCKER_IMAGE` &nbsp; &nbsp; &nbsp; &nbsp;                                      | The specific PMM Server Docker image for this guide.</br></br>Example: `perconalab/pmm-server-fb:PR-3251-a24d4f4`
+
+
+??? example "Expected output"
+        
+    ```{.text .no-copy}
+    export CH_HOST_IP=17.10.1.2
+    export VM_HOST_IP=17.10.1.3
+    export PG_HOST_IP=17.10.1.4
+    export PG_USERNAME=pmmuser
+    export PG_PASSWORD=pgpassword
+    export GF_USERNAME=gfuser
+    export GF_PASSWORD=gfpassword
+    export PMM_ACTIVE_IP=17.10.1.5
+    export PMM_ACTIVE_NODE_ID=pmm-server-active
+    export PMM_PASSIVE_IP=17.10.1.6
+    export PMM_PASSIVE_NODE_ID=pmm-server-passive
+    export PMM_PASSIVE2_IP=17.10.1.7
+    export PMM_PASSIVE2_NODE_ID=pmm-server-passive2
+    export PMM_DOCKER_IMAGE=perconalab/pmm-server-fb:PR-3251-a24d4f4
+    ```
+
+!!! note alert alert-primary "Note"
+    Ensure that you have all the environment variables from Step 1 set in each instance where you run these commands.
+
+### **Step 2: Create Docker network (Optional)**
+        
+1. Set up a Docker network for PMM services if you plan to run all the services on the same instance. As a result of this Docker network, your containers will be able to communicate with each other, which is essential for the High Availability (HA) mode to function properly in PMM. This step may be optional if you run your services on separate instances.
+
+2. Run the following command to create a Docker network:
+
+    ```sh
+    docker network create pmm-network --subnet=17.10.1.0/16
+    ```
+
+### **Step 3: Set up ClickHouse**
+
+ClickHouse is an open-source column-oriented database management system. In PMM, ClickHouse stores Query Analytics (QAN) metrics, which provide detailed information about your queries.
+
+To set up ClickHouse:
+
+1. Pull the ClickHouse Docker image.
+
+    ```sh
+    docker pull clickhouse/clickhouse-server:23.8.2.7-alpine
+    ```
+
+2. Create a Docker volume for ClickHouse data.
+
+    ```sh
+    docker volume create ch_data
+    ```
+
+3. Run the ClickHouse container.
+
+    === "Run services on same instance"
+
+        ```sh
+        docker run -d \
+        --name ch \
+        --network pmm-network \
+        --ip ${CH_HOST_IP} \
+        -p 9000:9000 \
+        -v ch_data:/var/lib/clickhouse \
+        clickhouse/clickhouse-server:23.8.2.7-alpine
+        ```
```

Comment:
@rnovikovP what do you think, should we mention an option to run all services on the same instance?

Reply:
I think it makes no sense for public documentation

Reply:
Hi all - my opinion:   
- Please keep the instructions on how to run the services all in one machine.  This is super useful for testing purposes and adoption. People might first try things locally.  And get familiar. And then add more complexity to their environment by adding new machines and splitting the services into the different machines.  But for testing purposes, good to have commands on how to start everything into a single machine.  We can add a warning that running all services in a single machine is not recommended for prod, but only development.  Thanks!

Reply:
+1 to keep the instructions for "single machine" as this should be a simple way for users to "start HA with a success"

Reply:
Added a note.